AntonEmery/array-test-template.js

```diff
@@ -0,0 +1,105 @@
+// Here's the old, simple assertion:
+function assert(claim,message) {
+    if (!claim) console.error(message);
+}
+
+// Here's a fancy new assertion, with better failure reporting:
+function expectValue(result, expected, attemptStr) {
+	if (result !== expected) {
+		console.log(attemptStr+" expected result "+expected+", got "+result);
+	}
+}
+// In your tests below, you may use either kind of assertion, or write your own!
+
+var testArray = [];
+
+function testPush(array) { // accept an array to test, if provided
+	// otherwise make a new one:
```

Comment:
You don't really need (or even want) to take in an array  for testing, since your tests are expecting the array to only have the values ['a', 'b', 'c']. Otherwise, looks good!